<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detailShownote</class>
 <widget class="QDialog" name="detailShownote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detail Show Note</string>
  </property>
  <widget class="QTextEdit" name="content_textEdit_detailshownote">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>731</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton_detailshownote">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>550</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Zurück</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_detailshownote">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>604</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="passwordProtectionCheckBox_detailshownote">
   <property name="geometry">
    <rect>
     <x>646</x>
     <y>15</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Notiz geschützt?</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel_detailshownote">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
